assets/js/modules/reader-revenue-manager/datastore/publications.js

```diff
@@ -68,35 +68,38 @@ const baseActions = {
 	 */
 	*syncPublicationOnboardingState() {
 		const registry = yield commonActions.getRegistry();
-		const connected = yield commonActions.await(
-			registry
-				.resolveSelect( CORE_MODULES )
-				.isModuleConnected( READER_REVENUE_MANAGER_MODULE_SLUG )
-		);
```

Comment:
I feel this is redundant here as we already do this check as part of `maybeSyncPublicationOnboardingState` which is the only place that eventually dispatches this action.